charts/values.yaml

```diff
@@ -21,6 +21,11 @@ clusterDomain: cluster.local
 ## There is a limit of 64 characters for CN in the certificate
 allowLongHostnames: false
 
+## This is flag to help upgrade from 1.0.x version of the chart to keep the use of legacy hostnames
```

Comment:
Sounds a bit confusing. How about this variant:
This flag facilitates upgrading from version 1.0.x of the chart while retaining the use of legacy hostnames. When upgrading from version 1.0.x to any version above 1.1.0, this flag must be set to true. It should remain true for all future upgrades. For new installations, this flag should be set to false.

Reply:
This version looks better and I will make a commit with this version.